cli/main.go

```diff
@@ -1,9 +1,23 @@
 package main
 
 import (
-	"github.com/kubeshop/tracetest/cli/cmd"
+	"fmt"
+	"os"
+
+	"github.com/kubeshop/tracetest/cli/global"
+	global_register "github.com/kubeshop/tracetest/cli/global/register"
 )
 
+func Execute() {
+	root := global.NewRoot()
+	global_register.Register(root)
```

Comment:
On this command we register all other commands, right? It makes sense to centralize all command registrations in one single file, just to see all commands that the CLI has?